certificate-analyser/README.md

```diff
@@ -0,0 +1,46 @@
+##Certificate validation tool
+
+### Usage
+
+java -jar certificate-analyser-<version>.jar --help
+```
+Usage: <main class> [-hl] [-kp[=<keyPasswd>]] [-tp[=<trustPasswd>]]
+                    [-a=<keyAlias>] [-k=<keyStore>] [-kt=<keyStoreType>]
+                    [-r=<remoteUrl>] [-t=<trustStore>] [-tt=<trustStoreType>]
+  -a, --keyalias=<keyAlias>
+                Alias under which this key is stored
+  -h, --help    display a help message
+  -k, --keystore=<keyStore>
```

Comment:
I think it would be good to document or check, that when somebody wants to use keyrings and configures the `safkeyring://` protocol, he has to add the `-Djava.protocol.handler.pkgs=com.ibm.crypto.provider` parameters to be able to access them

Reply:
I will update, thanks